Flick/Assets.xcassets/addFriendButtonDisabled.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "filename" : "add friend button disabled 1x.png",
```

Comment:
are we going to keep this convention for file names?

Reply:
should be camelCase to be consistent haha